models/experimental/functional_unet/tt/unet_shallow_ttnn.py

```diff
@@ -462,6 +462,7 @@ def __call__(self, x, move_input_tensor_to_device=True):
         ttnn.deallocate(c4_residual)
         x = self.upblock2(x, c3_residual)
         ttnn.deallocate(c3_residual)
+        c2_residual = ttnn.to_memory_config(c2_residual, ttnn.L1_MEMORY_CONFIG)
```

Comment:
Is this the reason for regression on unet perf?

Reply:
yes it is, I don't have bandwidth to dig into why is this helping